docs/open-source.mdx

```diff
@@ -5,50 +5,45 @@ sidebar_label: Overview
 ---
 
 Ory has an open source architecture, attitude, and heart. We believe in collaboration, participation, and the open exchange of
```

Comment:
```suggestion
Ory has an open-source architecture, attitude, and heart. We believe in collaboration, participation, and the open exchange of
```

Can you please go over the docs you edited and make sure to hyphenate "open source" when it's used as an adjective? -> "open-source NOUN"

Reply:
done. 

just a nitpick but
i would rather __always__ write "open source". It does not add to clarity IMO if it is sometimes with hyphen, you read and understand it the same either way...